### DISCUSSION TITLE
Could Image squence be loaded in image texture like in eevee?

### DISCUSSION THREAD

**Discussion by HSDSZ** [bnpr/Malt#534] [Q&A]
I rendered an animated character into image sequence. I hope to reload them into image texture and play the images repeatedly to release my PC viewport loading. But malt image texture does not support image sequence like in eevee. Is there any workaround?

**Comment by HSDSZ** [comment]
It is almost impossible to get a decent viewport fps when many fully rigged character moving in  the same scene.

**Comment by pragma37** [comment]
AFAIK, from the [last time I checked](https://github.com/bnpr/Malt/pull/348#issuecomment-1206602374), this can't be done from an addon.

The only built-in alternative is the Flipbook node, but it might not be suitable for your use case, depending on the required length and resolution of the sequence.
